### ISSUE TITLE
Fixing Umlauts in Summary

### ISSUE THREAD

**Issue by saigkill** [EdiWang/Moonglade#833]
**Describe the bug**
In the german language we using umlauts like (ä,ö,ü).
The Summary codes the umlauts as html code.

**To Reproduce**
1. Write an article in german
2. Let the summary field free
3. Automatic summary happens

**Expected behavior**
Better using the umlauts. Maybe using Unicode?

**Screenshots**
<img width="477" alt="image" src="https://github.com/user-attachments/assets/e4d816b1-126c-4c70-8108-83bcd19e8a68">

**Environment (please complete the following information):**
 - OS: Linux Docker
 - Browser Edge
 - Version latest prod

**Comment by EdiWang** [comment]
Could you give me an example German content that will trigger this issue? This is likely to be related to the encoder only added US and Asia language unicode ranges.

**Comment by EdiWang** [comment]
I am able to reproduce the issue using German text below:

```
In einer kürzlich veröffentlichten Studie wurde festgestellt, dass immer mehr Verbraucher nachhaltige Produkte bevorzugen. Die Untersuchung, die von einer führenden Universität durchgeführt wurde, zeigt, dass über 70% der Befragten angeben, regelmäßig umweltfreundliche Alternativen zu wählen.

„Es ist erfreulich zu sehen, dass das Bewusstsein für Nachhaltigkeit wächst“, sagte Dr. Müller, einer der Hauptautoren der Studie. „Die Menschen sind bereit, mehr für Produkte zu zahlen, die weniger schädliche Auswirkungen auf die Umwelt haben.“

Besonders beliebt sind Produkte aus recycelten Materialien sowie solche, die ohne schädliche Chemikalien hergestellt werden. Auch die Nachfrage nach regionalen und biologischen Lebensmitteln steigt stetig.

Die Studie betont jedoch auch, dass es noch viele Herausforderungen gibt. „Unternehmen müssen transparenter werden und sicherstellen, dass ihre Produkte wirklich nachhaltig sind“, fügte Dr. Müller hinzu.
```

**Comment by EdiWang** [comment]
Fixed via #836
